### PR TITLE
feat : Auth Added

### PR DESCRIPTION
**Pull request by sanu29**
Signup and Login added

image
image

### CODE REVIEW

**Review by ritu9898 — APPROVED** [review]
Good to merge

**Review by sanjitsarkar — APPROVED** [review]
Code looks clean and easy to understand

File: src/pages/signup.jsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                                        else{
+                                            setErrorDetails("Password Should have atleast 8 characters")
+                                        }
+                                    }
+                                    else{
+                                        setErrorDetails("Fileds cannot be empty")
+                                    }
+                            
+                            }}
+                 >Submit</button>
```
Comment (bhushannaikaroskar) [inline src/pages/signup.jsx]: You can separate this logic in some handler function for easy readability

File: src/pages/homepageAction.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        nav.style.display = "none"
+
+    }
+    else{
+        nav.style.display = "flex"
+
+    }
+   
+})
+
```
Comment (bhushannaikaroskar) [inline src/pages/homepageAction.js]: hey saniya this is incorrect way of writing code in react better write this logic in the  component containing menu. I guess in Header.jsx file

**Review by bhushannaikaroskar — CHANGES_REQUESTED** [review]
Hey saniya reviewed your code. I have given some suggestion please do check and implemnet those changes. I will approve once you are done with these changes

File: src/pages/signup.jsx (hunk 1 above)
Comment (sanu29) [reply src/pages/signup.jsx]: Done Changes

File: src/pages/homepageAction.js (hunk 2 above)
Comment (sanu29) [reply src/pages/homepageAction.js]: Done changes

**Review by developer-anand — APPROVED** [review]
All Looks good, can be merged.

Small suggestion in style be consistent with spacing it will make your code  more readable.

**Pull request merged**